mccpp/box/box.hpp

```diff
@@ -0,0 +1,21 @@
+
+#pragma once
+
+#include <iostream>
+#include <Eigen/Dense>
+#include <vector>
+#include <math.h>
+
+class Box
+{
+  public:
+     std::vector<double> box_dims;
+   Eigen::MatrixXd coordinates;
+
+  double volume(void);
+
+  Eigen::MatrixXd coord_wrap(void);
+
+  std::vector<double> minimum_image_dist(std::vector<double> i_particle);
```

Comment:
Also this should return an `Eigen::VectorXd` which will be directly convertible into a numpy array.

Reply:
Well changing the return type actually isn't necessary. I don't know if it will actually speed up the code now that I think about it.